mod_sample/models.py

```diff
@@ -88,7 +89,7 @@ def __repr__(self):
         return '<Sample extra for {id}>'.format(id=self.sample_id)
 
     @property
-    def short_name(self, length=5):
+    def short_name(self, length=5):     # type: ignore
```

Comment:
Why ignore the type?

Reply:
As mentioned in the PR, these are ad-hoc solutions I've put in place. I'll be removing it in the following days as I fix #281